packages/synapse-react-client/src/components/row_renderers/Dataset_V2.tsx

```diff
@@ -0,0 +1,132 @@
+    this.handleLinkClick = this.handleLinkClick.bind(this)
+  }
+
+  public handleLinkClick =
+    (link: string) => (event: MouseEvent<HTMLButtonElement>) => {
+      event.preventDefault()
+      window.open(
+        `${PRODUCTION_ENDPOINT_CONFIG.PORTAL}Synapse:${link}`,
+        '_blank',
+      )
+    }
+
+  public render() {
+    const {
+      data,
+      schema,
+      selectColumns,
+      columnModels,
+      genericCardSchema,
+      queryVisualizationContext: { getColumnDisplayName },
+    } = this.props
+    const datasetName = data[schema.datasetName]
+    const summary = data[schema.summary]
+    const tumorType = getValueOrMultiValue({
+      columnName: 'tumorType',
+      value: data[schema.tumorType],
+      selectColumns,
+      columnModels,
+    }).str
+    const diseaseFocus = getValueOrMultiValue({
+      columnName: 'diseaseFocus',
+      value: data[schema.diseaseFocus],
+      selectColumns,
+      columnModels,
+    }).str
+    const id = data[schema.id]
+    const fundingAgency = data[schema.fundingAgency]
+    const fileCount = data[schema.fileCount]
+    const fileSize = calculateFriendlyFileSize(data[schema.fileSize])
+    const values = [
+      [
+        'FUNDER',
+        getValueOrMultiValue({
+          columnName: 'fundingAgency',
+          value: fundingAgency,
+          selectColumns,
+          columnModels,
+        }).str,
+      ],
+      ['SIZE', fileSize],
+      ['FILES', fileCount],
+    ]
+    const synapseLink: string = data[schema.synapseLink] || ''
+    const synapseLinkId: string =
+      synapseLink.match(SYNAPSE_ENTITY_ID_REGEX)?.[1] || ''
+
+    if (genericCardSchema && genericCardSchema.secondaryLabels) {
+      const { secondaryLabels } = genericCardSchema
+      for (let i = 0; i < secondaryLabels.length; i++) {
+        const columnName = secondaryLabels[i]
+        const value: any = data[schema[columnName]]
+        let columnDisplayName
+        if (value) {
+          columnDisplayName = getColumnDisplayName(columnName)
+          const keyValue = [columnDisplayName, value, columnName]
+          values.push(keyValue)
+        }
+      }
+    }
+
+    return (
+      <div className="SRC-portalCard SRC-typeDataset  ">
+        <div className="SRC-cardThumbnail">
+          <Utils.Icon type={DATASET} />
+          <div>{fileSize}</div>
+        </div>
+        <div className="SRC-cardContent SRC-dataset">
+          <div className="SRC-type">Dataset </div>
+          <div>
+            <h3>
+              <a
+                target="_self"
+                href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}Synapse:${id}`}
+              >
+                {datasetName}
+              </a>
+            </h3>
+          </div>
+          <div className="SRC-description-dataset">
+            <p className="SRC-description-text">{summary}</p>
+            <div className="SRC-cardAction">
+              {synapseLink && <AddToDownloadCart folderId={synapseLinkId} />}
+            </div>
```

Comment:
I think the CCKP Datasets feature actually uses GenericCard, so I think you want to integrate this new feature into that component, not create a new DatasetV2 card component. Or am I misunderstanding?